Sources/SwiftDocCUtilities/Action/Actions/PreviewAction.swift

```diff
@@ -84,6 +86,7 @@ public final class PreviewAction: Action, RecreatingContext {
     ///     is performed.
     /// - Throws: If an error is encountered while initializing the documentation context.
     public init(
+        host: String,
```

Comment:
Since this is a public initializer, providing a default value here would avoid a source breaking change for anyone who imports SourceDocCUtilities.